app/assets/stylesheets/helpers/_organisations.scss

```diff
@@ -156,53 +156,33 @@
 /* Crests */
 
 @mixin organisation-logo-image($crest) {
-  background: image-url($crest) no-repeat 8px 0;
+  background: image-url('crests/#{$crest}_18px.png') no-repeat 8px 0;
+  @include media-down(mobile) {
```

Comment:
Is it possible to swap these two, so the 13px crest is defined first and `@include media(tablet)` is used to increase the crest size for larger screens?

I'd like to remove `media-down` where we can.